src/inspector_agent.cc

```diff
@@ -1,6 +1,8 @@
 #include "inspector_agent.h"
 
 #include "inspector_io.h"
+#include "base-object.h"
+#include "base-object-inl.h"
```

Comment:
I think you can leave these out, they are indirectly included by `node_internals` anyway :)

Reply:
Doesn't look like that is the case, unfortunately.